prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+
+const firstElement = languages[0];
+console.log('the first element of the array is:', firstElement);
+
+const thirdElement = languages[2];
+console.log('the third element of the array is:', thirdElement);
+
+const length = languages.length;
+console.log('the length of the array is:', length);
+
+const lastElement = languages[5];
+console.log('the last element of the array is:', lastElement);
```

Comment:
We can grab the last element of any array without knowing how long it is like this.

```js
var lastElement = languages[length - 1];
```

Reply:
/facepalm
i was going too fast and took the "subtract 1" instruction literally, lol.
i know better! i wrote the correct way in my notes moments before doing it wrong >< lol